client/src/App.tsx

```diff
@@ -1,12 +1,19 @@
 import { Theme } from './Theme';
 import { Button } from './Components/Button/Button';
+import { TextCard } from './Components/TextCard/TextCard';
 
 function App() {
   return (
     <Theme>
-      <Button onClick={() => {}}>
-        Zobacz pełny opis
+      <Button
+        onClick={() => {
+          // eslint-disable-next-line no-console
+          console.log('test');
+        }}
```

Comment:
Tak jak w poprzedniej PR zamieńmy to na specjalną akcje

Reply:
a tych specjalnych akcji nie dodajemy tylko w plikach stories? tu mamy główny App.tsx

Reply:
Tak jest, sorki

Reply:
To w takim razie kompletnie to usuńmy stąd :D Wgl nic nie powinniśmy miec w App z tych komponentów :D